Processor.php

```diff
@@ -137,10 +137,20 @@ private function processRenamedValues(array $renameMap, array $actualParams)
         return $actualParams;
     }
 
-    private function getParams(array $expectedParams, array $actualParams)
+    private function getParams(array $config, array $expectedParams, array $actualParams)
     {
-        // Simply use the expectedParams value as default for the missing params.
         if (!$this->io->isInteractive()) {
+            if (isset($config['exception-when-missing'])) {
```

Comment:
Agree, I chose isset, as that is similar as keep-outdated. Recommend we change both them to !empty && Inline::parse !== false